bin/create-maintenance-release

```diff
@@ -0,0 +1,123 @@
+#!/bin/bash
+set -e
+
+# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - FUNCTIONS
+# require helper
+currentDir=$(node -e "console.log(require('path').dirname(require('fs').realpathSync('$BASH_SOURCE')))")
+source $currentDir/helper
+
+help () {
+  log ""
+  log "This script creates a maintenance release (for the upstream e.g. livingdocs-editor / livingdocs-server."
+  log ""
+  log "-----------------------------------------------------------------------------------------------------------------------"
+  log "Parameters"
+  log "-----------------------------------------------------------------------------------------------------------------------"
+  log "--help / -h               Show help"
+  log "--base-tag                Base tag where the maintenance branch will be branched from e.g. '--base-tag v55.0.0'"
+  log "--npm-auth-token          npm-auth-token with write access e.g. '--npm-auth-token 11111111-2222-3333-4444-555555555555'"
```

Comment:
I don't see it how this can be automated. You need a human trigger to decide when a release happens. There is no indicator in the sourcecode which can do that.